app/controllers/movies_controller.rb

```diff
@@ -1,3 +1,5 @@
 class MoviesController < ApplicationController
-  def index; end
+  def index
+    @movies = Movie.where(genre: [Movie::RAILS_GENRE_LIST])
```

Comment:
`[]`はなくてもだいじょうぶですよ。定数自体がarrayであるため。
```suggestion
    @movies = Movie.where(genre: Movie::RAILS_GENRE_LIST)
```